rllib/policy/eager_tf_policy.py

```diff
@@ -565,12 +566,15 @@ def _compute_action_helper(self, input_dict, state_batches, episodes,
 
         @with_lock
         @override(Policy)
-        def compute_log_likelihoods(self,
-                                    actions,
-                                    obs_batch,
-                                    state_batches=None,
-                                    prev_action_batch=None,
-                                    prev_reward_batch=None):
+        def compute_log_likelihoods(
+                self,
+                actions,
+                obs_batch,
+                state_batches=None,
+                prev_action_batch=None,
+                prev_reward_batch=None,
+                actions_normalized=None,
```

Comment:
Why need both regular action and actions normalized?

Reply:
It's just a boolean flag here telling the method, whether the incoming actions are already normalized (between -1.0 and 1.0) or not.
If not AND the policy works in a normalized space (config.normalize_actions=True, which is the default now), we have to first normalize the incoming actions before we calculate the log-prob. Otherwise, the calculations would be wrong.

It should be True by default, b/c normally, we would only pass in already normalized actions that were produced by this policy, but a user may use this method with different inputs.
Remember that in RLlib, this method is only used in the OPE calculators.